dev/devicelab/lib/framework/runner.dart

```diff
@@ -92,32 +94,54 @@ Future<TaskResult> runTask(
   }
 }
 
-Future<VMIsolateRef> _connectToRunnerIsolate(Uri vmServiceUri) async {
+class _ClientAndIsolate {
```

Comment:
I don't love the name, but having a rough time coming up with something better. Maybe just `Runner` or `RunnerClient`? The client/isolate are sort of incidental just for the mechanics of the call, but conceptually this is a mechanism for firing off calls to the runner's service extension interface. 

I sort of wonder if the naming issue is more of an abstraction issue -- this feels like a bit of an in-between abstraction like Pair. You could go all the way and name it `Runner` then rather than exposing `callServiceExtention`, expose the actual methods you care about `runnerReady` and `runTask`, basically a sort of cocoon service interface. Given that this is private API, up to you if you want to go that far in this patch, which is more about migration, but it's definitely worth adding a small doc comment for future visitors to this code.

Reply:
Changing it to `RunnerClient` and doing a quick refactor here.